src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -65,11 +65,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         _actualFont{ DEFAULT_FONT_FACE, 0, 10, { 0, DEFAULT_FONT_SIZE }, CP_UTF8, false },
         _touchAnchor{ std::nullopt },
         _cursorTimer{},
-        _lastMouseClick{},
+        _lastMouseClickTS{},

```

Comment:
I don't recognize what a "TS" is straight off the bat. Rename it? (I had to search down to find it was a "timestamp".)